webauthn4j-util/src/main/java/com/webauthn4j/util/KeyUtil.java

```diff
@@ -29,6 +29,8 @@
  */
 public class KeyUtil {
 
+    private static SecureRandom secureRandom = new SecureRandom();
```

Comment:
make this final

Reply:
Overlooked your comment.
Will be fixed in #86